src/main/webapp/creatives/maps/simid_map.html

```diff
@@ -1,13 +1,14 @@
 <html>
   <head>
-    <link rel="stylesheet" type="text/css" href="simid_maps.css">
+    <link rel="stylesheet" type="text/css" href="simid_map.css">
```

Comment:
did you pull the most recent html because these changes were already merged to master?

Reply:
I did, when I compared to master in this PR it still had the old function names so I left that commit in.